### PR TITLE
adding mission timeline

### PR DESCRIPTION
**Pull request by Courey**
image

### CODE REVIEW

File: app/routes/labs.missions/route.tsx [hunk 1]
```diff
@@ -75,6 +76,29 @@ export default function () {
             2019 Astrophysics Senior Review, Executive Summary, Page 2
           </figcaption>
         </figure>
+        <div className="clearfix" />
```
Comment (lpsinger) [inline app/routes/labs.missions/route.tsx:79]: What does this do?

File: app/routes/labs.missions/route.tsx [hunk 2]
```diff
@@ -12,6 +12,7 @@ import { Meatball } from '~/components/meatball/Meatball'
 import type { BreadcrumbHandle } from '~/root/Title'
 
 import fleet from './AstroFleetChart.png'
+import missionTimeline from './mission-timeline.png'
```
Comment (lpsinger) [inline app/routes/labs.missions/route.tsx:15]: Where is the source artwork for this graphic? Is it an Adobe Illustrator file? A Python/matplotlib script? Can we get the source artwork under version control, please?

File: app/routes/labs.missions/route.tsx [hunk 3]
```diff
@@ -75,6 +76,29 @@ export default function () {
...
+        <div className="clearfix" />
+        <h3>Multimessenger Mission Timeline</h3>
+        <p className="usa-paragraph">
+          NASA missions played a critical role in the discovery and
+          characterization of the first binary neutron star merger (GW170817).
+          In the near future, the balanced mission portfolio is well-positioned
+          to continue to make major contributions to EM followup of
+          gravitational-wave sources. Current workhorse facilities in the area
+          of time-domain and multi-messenger astronony such as Fermi, Swift,
+          Chandra, & HST are well past design lifetimes. These capabiltiies are
```
Comment (lpsinger) [inline app/routes/labs.missions/route.tsx:88]:
```suggestion
          Chandra, & HST are well past design lifetimes. These capabilities are
```

Also, are ampersands allowed in text in JSX, or does this need to be written as `&amp;`?

File: app/routes/labs.missions/route.tsx [hunk 4]
```diff
@@ -75,6 +76,29 @@ export default function () {
...
+          currently being augemted and will eventually be replaced by specalized
+          CubeSats, SmallSats, and Missions of Opportunity, many of which are
+          currently in development and lager missions currently being proposed.
+        </p>
+        <figure>
+          <img
+            alt="a gantt chart depicting various mission timelines from the year 2000 to 2032"
```
Comment (lpsinger) [inline app/routes/labs.missions/route.tsx:95]:
```suggestion
            alt="Gantt chart depicting various mission timelines from the year 2000 to 2032"
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/labs.missions/route.tsx (hunk 2 above)
Comment (Courey) [reply app/routes/labs.missions/route.tsx:15]: I haven't the faintest idea. I took the image from the mock up. @dankocevski do you have any idea how the image is generated?

File: app/routes/labs.missions/route.tsx (hunk 3 above)
Comment (Courey) [reply app/routes/labs.missions/route.tsx]: it works fine as &
Screenshot 2024-01-05 at 2 37 08 PM

Comment (lpsinger) [reply app/routes/labs.missions/route.tsx]: I agree. And I see that the SSR code that is being sent to the browser contains `&amp;`.

File: app/routes/labs.missions/route.tsx (hunk 2 above)
Comment (lpsinger) [reply app/routes/labs.missions/route.tsx:15]: I'm going to open a new issue to get the source material for this graphic under version control. That way we can get this content onto the main branch.

**Review by lpsinger — APPROVED** [review]

File: app/routes/labs.missions/route.tsx (hunk 2 above)
Comment (Courey) [reply app/routes/labs.missions/route.tsx:15]: https://github.com/nasa-gcn/gcn.nasa.gov/pull/1839/files I have this branch. commenting here for posterity. It has the image, but it's just pulled from the existing mock up website.

**Pull request merged**